packages/api/tests/endpoints/test-granules.js

```diff
@@ -2139,6 +2250,69 @@ test.serial('PUT replaces an existing granule in all data stores with correct ti
   t.is(actualPgGranule.updated_at.getTime(), updatedEsRecord.updatedAt);
 });
 
+test.serial('PUT replaces an existing granule in all datastores with a granule that violates message-path write constraints, ignoring message write constraints and field selection', async (t) => {
```

Comment:
`a granule that violates message-path write constraints` Does it violate the message-path write constraints by updating a granule from completed to running?

Reply:
Correct - given the same execution.